src/app/routes/utils/regex/index.test.js

```diff
@@ -100,34 +100,31 @@ describe('articleDataPath', () => {
   shouldNotMatchInvalidRoutes(invalidRoutes, articleDataPath);
 });
 
-describe('frontPagePath', () => {
+describe('homePagePath', () => {
   const invalidRoutes = [
     '/news/home',
     '/persian/c5jje4ejkqvo.amp',
     '/iplayer',
     '/news/foobar',
     '/news/foobar.amp',
-    '/kyrgyz',
     '/kyrgyz.amp',
-    '/ukchina/simp',
-    '/ukchina/trad',
     '/serbian/lat.amp',
     '/serbian/cyr.amp',
   ];
-  shouldNotMatchInvalidRoutes(invalidRoutes, frontPagePath);
+  shouldNotMatchInvalidRoutes(invalidRoutes, homePagePath);
 });
 
-describe('frontPageDataPath', () => {
+describe('homePageDataPath', () => {
   const invalidRoutes = [
     '/news/data.json',
     '/iplayer.json',
     '/news/foobar.json',
     '/persian/.json',
-    '/persian.json',
-    '/serbian/cyr.json',
-    '/ukchina/trad.json',
+    // '/persian.json',
+    // '/serbian/cyr.json',
+    // '/ukchina/trad.json', // work out if we need these data routes still. They work locally but not live
```

Comment:
Hmm it's a good question. I believe that they are now valid homepage routes, since we have removed the frontPage routing. Let's move these into another test for valid routes.